data/mods/Xedra_Evolved/procgen/dreamsmith_procgen.json

```diff
@@ -75,96 +75,6 @@
         "increment": 1,
         "power_per_increment": 1,
         "ench_has": "WIELD"
-      },
-      {
```

Comment:
What I'll probably do here is make copy-from's of each of the dreamsmith weapons and then see about adding an eoc inside the recipe that selects "of burning" "glistening (acid) etc. So now all (most?) dreamsmith weapons will have something damage wise and it won't cost power points.

Reply:
I don't think it's necessary, there are still plenty of enchantments left that are still useful